src/main/java/com/redhat/cloud/policies/app/rest/PolicyCrudService.java

```diff
@@ -240,6 +242,104 @@ public Response getPoliciesForCustomer() {
+                  in = ParameterIn.QUERY,
+                  description = "Operations used with the filter",
+                  schema = @Schema(
+                          type = SchemaType.STRING,
+                          enumeration = {
+                                  "equal",
+                                  "like",
+                                  "ilike",
+                                  "not_equal",
+                                  "boolean_is"
+                          },
+                          defaultValue = "equal"
+                  )
+          ),
+          @Parameter(
+                  name = "filter[description]",
+                  in = ParameterIn.QUERY,
+                  description = "Filtering policies by the description depending on the Filter operator used.",
+                  schema = @Schema(type = SchemaType.STRING)
+          ),
+          @Parameter(
+                  name="filter:op[description]",
+                  in = ParameterIn.QUERY,
+                  description = "Operations used with the filter",
+                  schema = @Schema(
+                          type = SchemaType.STRING,
+                          enumeration = {
+                                  "equal",
+                                  "like",
+                                  "ilike",
+                                  "not_equal",
+                                  "boolean_is"
+                          },
+                          defaultValue = "equal"
+                  )
+          ),
+          @Parameter(
+                  name = "filter[is_enabled]",
+                  in = ParameterIn.QUERY,
+                  description = "Filtering policies by the is_enabled field, depending on the Filter operator used.",
+                  schema = @Schema(type = SchemaType.STRING)
+          ),
+          @Parameter(
+                  name="filter:op[is_enabled]",
+                  in = ParameterIn.QUERY,
+                  description = "Operations used with the filter",
+                  schema = @Schema(
+                          type = SchemaType.STRING,
+                          enumeration = {
+                                  "equal",
+                                  "like",
+                                  "ilike",
+                                  "not_equal",
+                                  "boolean_is"
+                          },
+                          defaultValue = "equal"
+                  )
+          ),
+  })
+  @APIResponse(responseCode = "400", description = "Bad parameter for sorting was passed")
+  @APIResponse(responseCode = "404", description = "No policies found for customer")
+  @APIResponse(responseCode = "403", description = "Individual permissions missing to complete action")
+  @APIResponse(responseCode = "200", description = "PolicyIds found", content =
+                 @Content(schema = @Schema(implementation = List.class)),
```

Comment:
I think this should be a PagedResponse (looking at what is being returned), otherwise just return the List below.

Reply:
I am not sure I follow. You mean to return a paged list to be consistent with getPolicies?

Reply:
No. I mean that it currently returns a PagedResponse (see below: `return PagingUtils.responseBuilder(page).build();
`). I think a `List` is better, as we don't paginate on this method, but that's not what this method is returning.

```
$  curl 'https://ci.foo.redhat.com:1337/api/policies/v1.0/policies/ids?offset=0&limit=50' # etc
{"data":["31695b91-9c45-4162-8aa5-b7ef5e9ea23b","d7073ec1-2e11-4353-91ad-06d5865b6c65","9f1b9a76-de92-4ee9-979e-b548b65c40d4"],"links":{"last":"/api/policies/v1.0/policies?limit=50&offset=0","first":"/api/policies/v1.0/policies?limit=50&offset=0"},"meta":{"count":3}}
```

Reply:
Now I get it. I think I was looking at the wrong place and thus misunderstood. Will change that later and also for enable/disable + delete to simple lists